src/test/java/teammates/test/cases/webapi/SaveFeedbackQuestionActionTest.java

```diff
@@ -269,6 +269,32 @@ public void testExecute_invalidQuestionNumber_shouldThrowException() {
                 logic.getFeedbackQuestion(typicalQuestion.getId()).getQuestionDescription());
     }
 
+    @Test
+    public void testExecute_invalidRecommendedLength_shouldThrowException() {
```

Comment:
Just a note. Test case should be written in `FeedbackTextQuestionDetailsText` rather than here. This is integration test which mainly test the interaction between components.

Reply:
Add this (what @xpdavid said) as a TODO